### ISSUE TITLE
API is returning 404 and requiring environmentId field...

### ISSUE THREAD

**Issue by ninjadev101** [novuhq/novu#471]
**Describe the bug**
`GET http://localhost:3000/v1/integrations 400 (Bad Request)`

**To Reproduce**
1. `pnpm start`

**Expected behavior**
Pulling integration array from `/v1/integrations`

**Screenshots**
![image](https://user-images.githubusercontent.com/52289096/163086955-98e58683-4446-4053-94fb-52056dbc0cab.png)

**Comment by scopsy** [comment]
Thanks for reporting @ninjadev101, could you please prefetch from `main` and reload the application? We've just merged the infra to support multiple #468 . If that doesn't work you can just clear your local DB and create another local user. 

If you need to keep the local data, you can create a migration in the following way (we will release final migration guide once the version will be ready)
1) Rename `applications` collection in mongo to be `environments`
2) Run the following commands in robomongo or similar:
```
db.getCollection('notificationtemplates').update({}, { $rename: {_applicationId: '_environmentId' }}, { multi: true });
db.getCollection('integrations').update({}, { $rename: {_applicationId: '_environmentId' }}, { multi: true });
db.getCollection('logs').update({}, { $rename: {_applicationId: '_environmentId' }}, { multi: true });
db.getCollection('messagetemplates').update({}, { $rename: {_applicationId: '_environmentId' }}, { multi: true });
db.getCollection('messages').update({}, { $rename: {_applicationId: '_environmentId' }}, { multi: true });
db.getCollection('notificationgroups').update({}, { $rename: {_applicationId: '_environmentId' }}, { multi: true });
db.getCollection('notifications').update({}, { $rename: {_applicationId: '_environmentId' }}, { multi: true });
db.getCollection('subscribers').update({}, { $rename: {_applicationId: '_environmentId' }}, { multi: true });
```

**Comment by ninjadev101** [comment]
Worked like charm!!
